Task_functions/index.js

```diff
@@ -1,11 +1,15 @@
 //Write code for the 1st task below this
-
-
+function addTWo(num1,num2){
+    return num1+num2;
+}
```

Comment:
There is an unnecessary space before the closing backtick in the return statement, it should be removed.